doc/api/n-api.md

```diff
@@ -2889,25 +2889,24 @@ Returns `napi_ok` if the API succeeded.
 This method is used within a callback function to retrieve details about the
 call like the arguments and the `this` pointer from a given callback info.
 
-### *napi_is_construct_call*
+### *napi_get_new_target*
 <!-- YAML
-added: v8.0.0
+added: v9.0.0
```

Comment:
I don't know if it's worth it since n-api is still unofficial, but we could also make a function level changelog entry stating that it was renamed.